Pull-request 19 review (#21)
materials-marketplace/user-docs: docs/source/apps/mp-api.md

# Standard API

> The MarketPlace defines a standard API through which the registered application can be communicated and monitored via the MarketPlace.

Reviewer: ```suggestion
The MarketPlace defines a standard API through which the registered application can be communicated with and monitored via the platform.
```

> The API is defines a set of of endpoints or _capabilities_ to specify actions which MarketPlace applications may perform.

Reviewer: ```suggestion
The API defines a set of of endpoints or _capabilities_ to specify actions which MarketPlace applications may perform.
```

An application provider may choose to support any subset of these capabilities, depending on the relevant functionalities.

> The complete specification is hosted publicly on GitHub:<br>

Reviewer: ```suggestion
The complete specification is hosted publicly on GitHub:

```

Author: I think it actually looks better with the line break.

Reviewer: Looks like there might be other options, but [this is the way to go ](https://www.markdownguide.org/basic-syntax/#line-break-best-practices)

> [https://github.com/materials-marketplace/standard-app-api](https://github.com/materials-marketplace/standard-app-api).<br>

Reviewer: ```suggestion
[https://github.com/materials-marketplace/standard-app-api](https://github.com/materials-marketplace/standard-app-api).

```

Author: I think it actually looks better with the line break.

Reviewer: But you should be able to do it without HTML code, should't you? That is what I was trying to achieve.

Author: I think it doesn't work without

Reviewer: [Indeed](https://github.com/materials-marketplace/user-docs/pull/21#discussion_r938806995).

You can also browse the official specification with Redoc <a href="api.html" target="_blank">here</a>.

> Below is more information about the API specifications. [Here](../capabilities) more about app capabilities. Once completed, the application can be registered to the platform, as explained in [this](registration.md) page.

Reviewer: ```suggestion
Below is more information about the API specification. [Here](../capabilities), more about app capabilities. Once completed, the application can be registered on the platform, as explained in [this page](registration.md).
```

Author: Some changes I adopted and adapted.
Concerning the "register on", I think "to" is more accurate because that's what you are registering to, if that makes sense.

Reviewer: I think after "to" you usually add a verb. It is meant for actions, I believe. But I'm not sure what would be the best option.

Author: Not always: "He goes to _school_".

Reviewer: I mean in case of [register](https://ell.stackexchange.com/a/159339). But I don?t think _for_ is the answer either.

Author: I'm still in favor of "register to" :)
